satellitevu/apis/orders.py

```diff
@@ -177,3 +192,202 @@ def _save_order_to_zip(self, destdir: str, order_id: UUID, item_ids: List[str]):
+            item_id: A string representing the specific image identifiers e.g.
+            "20221010T222611000_basic_0_TABI".
+
+        Returns:
+            A dictionary containing the url which the image can be downloaded from.
+        """
+        url = self.url(f"/{contract_id}/{order_id}/{item_id}/download?redirect=False")
+
+        response = self.make_request(method="GET", url=url)
+        return response.json()
+
+    def download_item(
+        self,
+        *,
+        contract_id: Union[UUID, str],
+        order_id: UUID,
+        item_id: str,
+        destdir: str,
+    ) -> str:
+        """
+        Download a submitted imagery order.
+
+        Args:
+            contract_id: String or UUID representing the ID of the Contract
+            which an item in the order is associated with.
+
+            order_id: UUID representing the order id e.g.
+            "2009466e-cccc-4712-a489-b09aeb772296".
+
+            item_id: A string representing the specific image identifiers e.g.
+            "20221010T222611000_basic_0_TABI".
+
+            destfile: A string (file path) representing the directory to which
+            the imagery will be downloaded.
+
+        Returns:
+            A string specifying the path the imagery has been downloaded to.
+
+        """
+        item_url = self.item_download_url(
+            contract_id=contract_id, order_id=order_id, item_id=item_id
+        )["url"]
+
+        response = self.make_request(method="GET", url=item_url)
+
+        destfile = os.path.join(destdir, f"{item_id}.zip")
+        data = raw_response_to_bytes(response)
+
+        return bytes_to_file(data, destfile)
+
+    def download_order(
+        self,
+        *,
+        contract_id: Union[UUID, str],
+        order_id: UUID,
+        destdir: str,
+    ) -> str:
+        """
+        Downloads entire imagery order into one ZIP file.
+
+        Args:
+            contract_id: String or UUID representing the ID of the Contract
+            which an order is associated with.
+
+            order_id: UUID representing the order id e.g.
+            "2009466e-cccc-4712-a489-b09aeb772296".
+
+            destdir: String specifying path of the directory where imagery will
+            be downloaded to
+
+        Returns:
+            A string specifying the path the imagery has been downloaded to.
+            All items will be downloaded into one ZIP file.
+        """
+        order_details = self.get_order_details(
+            contract_id=contract_id, order_id=order_id
+        )
+
+        order_id = order_details["id"]
+        item_ids = [i["properties"]["item_id"] for i in order_details["features"]]
+
+        return self._save_order_to_zip(destdir, contract_id, order_id, item_ids)
+
+    def _save_order_to_zip(
+        self,
+        destdir: str,
+        contract_id: Union[UUID, str],
+        order_id: UUID,
+        item_ids: List[str],
+    ):
+        destzip = os.path.join(destdir, f"SatelliteVu_{order_id}")
```

Comment:
Do we want to change the destination directory to `SatVu` for v2?

Reply:
yeah good shout